src/core/lv_obj.c

```diff
@@ -59,6 +63,13 @@ const lv_obj_class_t lv_obj_class = {
     .group_def = LV_OBJ_CLASS_GROUP_DEF_FALSE,
     .instance_size = (sizeof(lv_obj_t)),
     .base_class = NULL,
+#if LV_USE_OBJID_BUILTIN
+    .obj_count = &obj_count,
```

Comment:
`obj_count` could be useful for custom `lv_obj_assign_id`, let's the config to `#if LV_USE_OBJID` ?